tools/db_bench_tool.cc

```diff
@@ -3630,6 +3642,10 @@ class Benchmark {
       thread->stats.FinishedOps(nullptr, db, 1, kRead);
       ++i;
     }
+    if (thread->shared->read_rate_limiter.get() != nullptr) {
+      thread->shared->read_rate_limiter->Request(i, Env::IO_HIGH);
```

Comment:
Yes, does that mean that the whole penalty will be at the end ? do we care about having smooth penalty ?